docs/DeveloperGuide.md

```diff
@@ -451,108 +453,318 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+
+* 2a. The system detects an invalid contact index.
     * 2a1. The system displays an error message stating the contact index is invalid.
 
-        Use case resumes at step 2.
+        Use case resumes at step 1.
 
 
-* 2b. The system detects an email input error (invalid format):
+* 2b. The system detects an email input error (invalid format).
     * 2b1. The system displays an error message stating the correct format.
   
-        Use case resumes at step 2.  
+        Use case resumes at step 1.  
   
-  **Use case: UC06 Search for contacts by Name**
+
+**Use case: UC06 Search for contacts by Name**
 
 **MSS**
 
 1.  User searches for the contact by name.
-2.  WedLinker shows a list of contacts containing the name.
+2.  System shows a list of contacts containing the name.
 
 **Use case: UC07 Filter by Tag**
 
 **MSS**
 
 1. User filters for contacts with a specified tag.
-2. WedLinker only shows a list of contact with the specified tag.
+2. System only shows a list of contact with the specified tag.
 
    Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The list is empty.
 
   Use case ends.
 
+
 **Use case: UC08 Create Tags**
 
 **MSS**
 
-1. User creates a tag.
-2. WedLinker displays the successful creation of tag.
+1. User requests to create a tag.
+2. System displays the successful creation of tag.
 
    Use case ends.
 
 **Extensions**
 
-* 2a. The tag already exists.
-  * 2a1. WedLinker does not create a new tag.
-  * 2a2. WedLinker informs the user the tag already exists.
+* 1a. The tag already exists.
+    * 1a1. System does not create a new tag.
+    * 1a2. System informs the user the tag already exists.
+
+      Use case ends.
 
-    Use case ends.
 
 **Use case: UC09 Tagging a contact with a specified tag**
 
 **MSS**
 
-1. User searches for the contact to be tagged (UC06).
-2. WedLinker shows a list of contacts for the search.
-3. User adds the tag to the contact based on the index of the list.
-4. WedLinker informs the user the contact is tagged.
-5. WedLinker shows the user the final result of the contact.
+1. User <ins>lists all contacts (UC01)</ins>.
+2. User adds the tag to the contact.
+3. System informs the user the contact is tagged.
+4. System shows the user the final result of the contact.
 
    Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The list is empty.
 
   Use case ends.
 
-* 3a. WedLinker detects that the tag does not exist.
+* 2a. System detects that the tag does not exist.
+
+    * 2a1. System <ins>creates a new tag (UC08)</ins>
 
-    * 3a1. WedLinker creates a new tag (UC08)
+      Use case resumes at step 3.
 
-      Use case resumes at step 4
+* 2b. The given index is invalid.
 
-* 3b. The given index is invalid.
+    * 2b1. System shows an error message prompting the user to enter a valid index.
 
-    * 3b1. WedLinker shows an error message prompting the user to enter a valid index.
+      Use case resumes at step 1.
 
 **Use case: UC10 Delete Contact**
```

Comment:
Since only the first letter is capitalized in the other use cases maybe it should be

> **Use case: UC10 Delete contact**